src/slds.ts

```diff
@@ -0,0 +1,1397 @@
+// Custom suffix list is used instead of Public Suffix List https://publicsuffix.org/, so as not to increase the size of the package
```

Comment:
This list was derived from a source, so we should cite it:
```suggestion
// Custom suffix list is used instead of Public Suffix List https://publicsuffix.org/, so as not to increase the size of the package.
// List derived from https://github.com/gavingmiller/second-level-domains (Copyright (c) 2013 Gavin Miller, licensed under the MIT License).
```